package.json

```diff
@@ -8,8 +8,11 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "1.39.0",
+    "@playwright/test": "^1.47.2",
     "@types/node": "^20.9.3",
     "dotenv": "^16.3.1"
+  },
+  "dependencies": {
+    "playwright": "^1.46.1"
```

Comment:
Why did you add a playwright to "dependencies" and why this version is lower then in "devDependencies".
It think this import is not needed

Reply:
This was an accident, kind of.  I was struggling with my VSCode and reinstalled/updated, which probably added this.  I can revert it.  You're right, it's not necessary.